roles/oidc-op/tasks/main.yml

```diff
@@ -86,20 +87,28 @@
     owner: oidc-op
     group: oidc-op
     mode: 0644
+  # This file is checked out as root but needs to be
+  # writable by oidc-op
+  changed_when: false
```

Comment:
Haha, grapjas ;)
Wat dacht je ervan om gewoon in `config.json` een pad te kiezen dat niet wordt overschreven door een checkout?

Reply:
Sorry, niet aan gedacht. Ik ging er vanuit dat dit allemaal hard-coded files waren in de example/flask_op